pkg/ddl/tests/fail/fail_db_test.go

```diff
@@ -57,6 +60,10 @@ func createFailDBSuiteWithLease(t *testing.T, lease time.Duration) (s *failedSui
 			s.cluster = c
 		}),
 	)
+	if kerneltype.IsNextGen() {
+		testenv.UpdateConfigForNextgen(t)
+		kvstore.SetSystemStorage(s.store)
+	}
```

Comment:
I've changed this at #63192 as well, can I cherry-pick the rest of your changes to #63192 (to reduce the cherry-pick work to the preview branch `release-nextgen-20250815`)?

Reply:
Sure, less PR reduces the duplicate work